src/Queue/QueueServiceProvider.php

```diff
@@ -104,24 +104,6 @@ public function __call($name, $arguments)
         $this->app->alias(Factory::class, 'queue');
         $this->app->alias(Worker::class, 'queue.worker');
         $this->app->alias(Listener::class, 'queue.listener');
-
-        $this->registerCommands();
-    }
-
-    protected function registerCommands()
```

Comment:
Does this need to be replaced with something? @luceos you're a bit more familiar with the queue system than I am

Reply:
Yes, the commands need to be registered still.

Reply:
I've "tried" registering the commands. Let me know if the approach is okay or not :)